receiver/hostmetricsreceiver/internal/scraper/processscraper/process_scraper_test.go

```diff
@@ -78,6 +79,13 @@ func TestScrape(t *testing.T) {
 	assertSameTimeStampForAllMetricsWithinResource(t, resourceMetrics)
 }
 
+func assertSchemaIsSet(t *testing.T, resourceMetrics pdata.ResourceMetricsSlice) {
+	for i := 0; i < resourceMetrics.Len(); i++ {
+		rm := resourceMetrics.At(0)
+		assert.EqualValues(t, conventions.SchemaURL, rm.SchemaUrl())
+	}
+}
+
```

Comment:
Is an additional test to ensure the default doesn't override a SchemaURL that was set needed here?

Reply:
We don't have mock scapers, I would have to introduce one. I think it is not worth adding so much code that will require maintenance just to check a single `if` statement. Happy to do it if you think otherwise.